explorer/client/src/pages/object-result/ObjectLoaded.tsx

```diff
@@ -67,12 +64,6 @@ function ObjectLoaded({ data }: { data: DataType }) {
                 const singleOwnerPattern = /SingleOwner\(k#(.*)\)/;
```

Comment:
I believe we should also be able to address the TODO comments from line 48 to line 55 by leveraging the [ObjectOwner](https://github.com/MystenLabs/sui/blob/b180b663c0b755c97ea37ad57ff636fb04f2e158/sdk/typescript/src/types/common.ts#L9) in the SDK. Feel free to add some common util function in the SDK if needed